File: python/sample-authentication-code/tests/sendFirstRequestmTls_test.py
from sendFirstRequestmTls import send_first_request

def test_send_first_request_mtls():
    """Test the send_first_request function to ensure it sends a request with mtls authentication."""
    url = "https://apigatewaycat.jpmorgan.com/accessapi/balance"
    response = send_first_request(url)
    assert response.status_code == 200, f"Expected status code 200, got {response.status_code}"
    assert response.text, "Response text should not be empty"

File: go/sample-authentication-code/pkg/samples/sendFirstRequestMtls_test.go
package samples

import (
	"testing"
)

func TestSendFirstRequestMtls(t *testing.T) {
	url := "https://apigatewaycat.jpmorgan.com/accessapi/balance"
	status := SendFirstRequestMtls(url)
	if status != "200 OK" {
		t.Errorf("Expected status 200 OK, got %s", status)
	}
}
